### ISSUE TITLE
Xception pretrained

### ISSUE THREAD

**Issue by haibochina** [LikeLy-Journey/SegmenTron#23]
Thank you for your great job! Is there any pretrained model for DFANet?

**Comment by LikeLy-Journey** [comment]
i have not trained this model

**Comment by haibochina** [comment]
> i have not trained this model

I am training now!

**Comment by linbaiwpi** [comment]
> > i have not trained this model
> 
> I am training now!

How is your training right now? Using the pre-trained Xception or training from scratch?

**Comment by haibochina** [comment]
It is still training now！ The result without pretrained model is 59 miou&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by DePengW** [comment]
> 现在仍在训练！没有预训练模型的结果是59 miou＆nbsp; 发自我的iPhone
> […](#)
> ------------------ Original ------------------ From: Lin Bai <notifications@github.com&gt; Date: Sat,Mar 14,2020 9:57 AM To: LikeLy-Journey/SegmenTron <SegmenTron@noreply.github.com&gt; Cc: haibo <1974273017@qq.com&gt;, Author <author@noreply.github.com&gt; Subject: Re: [LikeLy-Journey/SegmenTron] Xception pretrained (#23) i have not trained this model I am training now! How is your training right now? Using the pre-trained Xception or training from scratch? — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by haliphinx** [comment]
@haibochina Hey, are you training the DFANet on the cityscape dataset? What's your final accuracy and what's the number of epoch in total to achieve that? 
I trained it from scratch, but it stuck into around 30 miou from epoch 100.
Thanks!!

**Comment by haibochina** [comment]
@haliphinx I just trained XceptionA the way that shufflenet use. and just trained 30000 iterarion, then I trainied DFANet on cityscape, the mIOU is 68.9.
[https://github.com/megvii-model/ShuffleNet-Series](url)

**Comment by haliphinx** [comment]
@haibochina Thanks for your reply. The accuracy sounds great! I'm wondering if it's possible to share your pretrained XceptionA model please.

**Comment by haibochina** [comment]
I have left the company that I intershiped, so ...
